packages/ra-auth-auth0/src/authProvider.ts

```diff
@@ -141,8 +142,7 @@ export const Auth0AuthProvider = (
                 await client.handleRedirectCallback();
                 return;
             } catch (error) {
-                console.log('error', error);
-                throw error;
+                throw { redirectTo: false };
```

Comment:
why not return also the error message, so that it can be displayed in the AuthCallback page?